--- etc/rc.local.orig	2015-02-12 15:09:51.087897267 +0100
+++ etc/rc.local	2015-02-12 15:16:14.567880971 +0100
@@ -11,4 +11,12 @@
 #
 # By default this script does nothing.
 
+hostkeys() {
+        /usr/sbin/dpkg-reconfigure openssh-server
+        for key in /etc/ssh/ssh_host_*.pub; do
+                ssh-keygen -lf $key
+        done
+}
+ls /etc/ssh/ssh_host_* > /dev/null || hostkeys
+
 exit 0
